modules/mediawiki/templates/jobrunner.json.erb

```diff
@@ -30,11 +30,12 @@
         },
         // runJobs.php process time limits
         "real": {
-            "*": 300
+            "*": 300,
+            "refreshLinks": 9000,
```

Comment:
far too high to have set on all runners. We could kill the queue with other backlogs building in this time